docs/Makefile

```
@@ -80,23 +79,27 @@ install-tools:
 	gem install html-proofer
 
 
-# use vx.y.z-latest as the tag for images that are serving version x.y.z of the docs under the docs.solo.io/gloo/latest/ path
-SERVE_AS_LATEST_TAG:=$(VERSION)-latest
+# use "current" instead of the default "latest" so we don't accidentally push something with this tag 
+CURRENT_TAG:=current
 
 .PHONY: docker-push-docs
 docker-push-docs: site-release manifest
+ifeq ($(ON_DEFAULT_BRANCH), true)
 	docker build \
 		--build-arg VERSION=latest \
 		--build-arg PRODUCT_SCOPE=$(PRODUCT_SCOPE) \
 		--build-arg FROM_DIR=./site-latest \
-		-t $(GCR_REPO_PREFIX)/gloo-docs:$(SERVE_AS_LATEST_TAG) .
-	docker push $(GCR_REPO_PREFIX)/gloo-docs:$(SERVE_AS_LATEST_TAG)
+		-t $(GCR_REPO_PREFIX)/gloo-docs:$(CURRENT_TAG) .
+	docker push $(GCR_REPO_PREFIX)/gloo-docs:$(CURRENT_TAG)
+endif
+ifeq ($(RELEASE),"true")
```

this check needs to have the parenthesis removed,
```suggestion
ifeq ($(RELEASE),true)
```

---

Is it different than in the parent Makefile? I was using what's done there as an example.

---

This is different because the variable is passed to the docs makefile via an env var, which evaluates & removes the quotes from the base makefile variable

---

You have to be very careful with quotes in bash/makefiles

---

Why does it have quotes in the original assignment?

---

We could remove them to be consistent everywhere. No good reason for the quotes in the first makefile, but you can see more general rules about quotes in makefiles here: https://stackoverflow.com/questions/23330243/gnu-makefile-how-to-and-when-to-quote-strings